zgw/openklant/docker-compose-override.yml

```diff
@@ -2,7 +2,7 @@ services:
 
   # openklant 2
   openklant-2:
-      image: maykinmedia/open-klant:2.7.0
+      image: maykinmedia/open-klant:latest
```

Comment:
please don't use `latest` here since we need to test against a specific version that we support from the libraries, especially in test scenarios. leave it at `2.8.0`